gslib/commands/mb.py

```diff
@@ -162,7 +169,13 @@
                          Valid values are "enforced" or "unspecified".
                          Default is "unspecified".
 
-""")
+  --rpo setting          Specifies the replication setting.
```

Comment:
Not sure about the possible "setting' values

Reply:
Done.